src/js/AirshipGimbalEventEmitter.ts

```diff
@@ -1,29 +1,35 @@
-// @flow
-'use strict';
-
 import {
   NativeModules,
   NativeEventEmitter,
-  Platform
-} from 'react-native';
-
-const AirshipGimbalAdapterModule = NativeModules.AirshipGimbalAdapterModule;
+  EmitterSubscription,
+  Platform,
+} from "react-native";
 
-class AirshipGimbalEventEmitter extends NativeEventEmitter {
+const { AirshipGimbalAdapterModule } = NativeModules;
 
+/**
+ * @ignore
```

Comment:
fwiw, while it's supposedly supported I don't think `@ignore` was working for me in my own testing. might be worth verifying.